src/requirements/EthereumAttestation/EthereumAttestationRequirement.tsx

```diff
@@ -0,0 +1,94 @@
+import { HStack, Img } from "@chakra-ui/react"
+import BlockExplorerUrl from "components/[guild]/Requirements/components/BlockExplorerUrl"
+import DataBlock from "components/[guild]/Requirements/components/DataBlock"
+import Requirement, {
+  RequirementProps,
+} from "components/[guild]/Requirements/components/Requirement"
+import { RequirementLinkButton } from "components/[guild]/Requirements/components/RequirementButton"
+import { useRequirementContext } from "components/[guild]/Requirements/components/RequirementContext"
+import { RPC } from "connectors"
+import useResolveAddress from "hooks/resolving/useResolveAddress"
+import shortenHex from "utils/shortenHex"
+
+export const EAS_SCAN_BASE = {
+  ARBITRUM: "https://arbitrum.easscan.org/schema/view",
+  OPTIMISM: "https://optimism.easscan.org/schema/view",
+  ETHEREUM: "https://easscan.org/schema/view",
+  SEPOLIA: "https://sepolia.easscan.org/schema/view",
+  BASE_GOERLI: "https://base-goerli.easscan.org/schema/view",
+} as const
+
+const EthereumAttestationRequirement = (props: RequirementProps): JSX.Element => {
+  const requirement = useRequirementContext()
+
+  const recipientDomain = useResolveAddress(requirement.data?.recipient)
+  const attesterDomain = useResolveAddress(requirement.data?.attester)
+
+  return (
+    <Requirement
+      image={
+        <Img
+          src={"/requirementLogos/eas.png"}
+          maxWidth={"var(--chakra-space-8)"}
+          maxHeight={"var(--chakra-space-8)"}
+        />
+      }
+      footer={
+        <HStack spacing={4}>
+          <RequirementLinkButton
+            href={`${EAS_SCAN_BASE[requirement?.chain ?? "ETHEREUM"]}/${
+              requirement?.data?.schemaId
+            }`}
+            imageUrl="/requirementLogos/eas.png"
+          >
+            {/* View schema on EAS scan */}
```

Comment:
I think you can remove these comments if you decide which copy should we use ;)

Reply:
I think the current version displaying only "Attester", "Recipient" or "Schema" is better, removed these comments in https://github.com/agoraxyz/guild.xyz/pull/873/commits/1a88792d9308c6a9971859a949ec5458a2558e51 👍